src/Storefront/Controller/CmsController.php

```diff
@@ -61,7 +61,7 @@ public function __construct(
      * @throws MissingRequestParameterException
      * @throws PageNotFoundException
      */
-    public function page(string $id, Request $request, SalesChannelContext $salesChannelContext): Response
+    public function page(?string $id, Request $request, SalesChannelContext $salesChannelContext): Response
```

Comment:
i guess you can better remove `if (!$id) {` and its `"id"=null` default value annotation

Reply:
The real @Haehnchen reviewed one of my PRs? 🙇 I will do as @keulinho likes